R/ResampleDesc.R

```diff
@@ -47,6 +48,13 @@
 #'     Default is 10.}
 #'   \item{folds [\code{integer(1)]}}{Folds in the repeated CV for \code{RepCV}.
 #'     Here \code{iters = folds * reps}. Default is 10.}
+#'   \item{horizon [\code{integer(1)}]}{Number of observations to forecast for \dQuote{GrowingWindowCV}
+#'    and \dQuote{FixedWindowCV}. Default is 1.}
+#'   \item{initial.window [\code{numeric(1)}]}{Fraction of observations to start with
```

Comment:
I would mention that this determines the training set.

Reply:
check